libs/spark/stories/card.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Meta } from '@storybook/react/types-6-0';
+import { Card, CardContent, CardMedia, Typography } from '@material-ui/core';
+import styled from 'styled-components';
+
+export default {
+  title: 'prenda-spark/Card',
+  component: Card,
+  argTypes: {},
+  args: {},
+} as Meta;
+
+export const BasicCard = () => (
+  <Card>
+    <CardContent>
+      <Typography gutterBottom variant="h6" component="h2">
+        Card Title
+      </Typography>
+      <Typography variant="body1" component="p">
+        Card body
+      </Typography>
```

Comment:
For each of these stories, I'd suggest copying the text/content in the figma pages. It doesn't matter technically, but just to validate the overall look with a bunch of text is nice.

Reply:
I updated both Card examples to match their Figma counterparts.